src/WebViewShared.tsx

```diff
@@ -16,29 +16,38 @@ import {
 import styles from './WebView.styles';
 
 const defaultOriginWhitelist = ['https://*'] as const;
+const defaultDeeplinkWhitelist = ['https://*'] as const;
```

Comment:
Also, this is a bit confusing because the `https` protocol is not a deep link.

Reply:
cc/ @ChALkeR

Reply:
> Also, this is a bit confusing because the https protocol is not a deep link.

In this context it is. `https:` is a registered deep link handler by Chrome or whatever app browser app.

Reply:
I don't have a strong opinion here. I believe it's not a blocker for this iteration. @ChALkeR, please let me know if you think this requires revision.